cmd/rookflex/cmd/init.go

```diff
@@ -36,7 +37,17 @@ func init() {
 }
 
 func initPlugin(cmd *cobra.Command, args []string) error {
-	fmt.Println(`{"status":"Success", "capabilities": {"attach": false}}`)
+	status := flexvolume.DriverStatus{
+		Status: flexvolume.StatusSuccess,
+		Capabilities: &flexvolume.DriverCapabilities{
+			Attach: false,
+			// Required for cephfs (ReadWriteMany)
+			SELinuxRelabel: false,
+		},
+	}
+	if err := json.NewEncoder(os.Stdout).Encode(&status); err != nil {
+		return err
+	}
 	os.Exit(0)
```

Comment:
shouldn't the `status` be printed out here?

Reply:
It's streamed to `os.Stdout` by the encoder at L48.